### ISSUE TITLE
Windows  declaracad-0.4.1-amd64 - "no shapes to display"

### ISSUE THREAD

**Issue by carlkl** [codelv/declaracad#52]
I installed declaracad-0.4.1-amd64.msi from CI and opened one of the example files. However, no shapes are shown and the log in te output window ends with: "no shapes to display"

**Comment by carlkl** [comment]
I tested again with https://github.com/codelv/declaracad/actions/runs/457915181, but the issue is still there.

[declaracad_win64.log](https://github.com/codelv/declaracad/files/5761767/declaracad_win64.log)

**Comment by frmdstryr** [comment]
Yeah for some reason the communication to the viewer isn't working on windows. I think it needs rewritten to use sockets...

**Comment by frmdstryr** [comment]
Thanks for giving it a try. The standalone viewer should work now at least.

**Comment by carlkl** [comment]
Yes, the view mode works, thanks.

And a good start in the new year for you!

**Comment by carlkl** [comment]
Just an idea:

mayby one can think about using [Zeromq](https://github.com/zeromq/pyzmq) or [Yoton](https://github.com/pyzo/pyzo/tree/master/pyzo/yoton) (_pure python_) instead of using sockets.

**Comment by frmdstryr** [comment]
Good idea,  zmq's should work well.
